### PR TITLE
🔀Merge "Working on" to "Master/main"

### PR DESCRIPTION
**Pull request by mathis150**
Add Exercice 2 with security rules of the repo.

### CODE REVIEW

**Comment by mathis150** [comment]
Test

File: ReadME.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# Git du projet "DevOPS" Ynov de Mathis LENOIR.
\ No newline at end of file
+# Git du projet "DevOPS" Ynov de Mathis LENOIR.
+
+### Sommaire :
+- TP1: Contient tout les exercices du dit Travail Pratique.
```
Comment (quentin-desbin) [inline ReadME.md]: Maybe you could add a link to the `TP1/ReadME.md` ?

File: ReadME.md
Comment (quentin-desbin) [inline ReadME.md:1]: File name should be `README.md`

File: TP1/ReadME.md
Comment (quentin-desbin) [inline TP1/ReadME.md:1]: File name should be `README.md`

**Review by quentin-desbin — CHANGES_REQUESTED** [review]

**Comment by mathis150** [comment]
Ready for new review @quentin-desbin.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Git du projet "DevOPS" Ynov de Mathis LENOIR.
+
+### Sommaire :
+- [TP1](TP1/ReadME.md): Contient tout les exercices du dit Travail Pratique.
```
Comment (quentin-desbin) [inline README.md:4]: Link is bad

Comment (quentin-desbin) [inline README.md]:
```suggestion
- [TP1](TP1/README.md): Contient tout les exercices du dit Travail Pratique.
```

**Review by quentin-desbin — CHANGES_REQUESTED** [review]

**Review by quentin-desbin — APPROVED** [review]

**Pull request merged**